test/unit/lint/test-dependencies.js

```diff
@@ -24,6 +24,52 @@ const native = [
 ];
 
 describe('dependencies', () => {
+  let appPackages;
+  let mainPackages;
+  let mainDevPackages;
+  let prodPackages;
+
+  beforeEach(async function() {
```

Comment:
There isn't much point in doing this before every test, maybe move it to the top-level?